elasticapm/instrumentation/packages/grpc.py

```diff
@@ -74,3 +76,30 @@ def call(self, module, method, wrapped, instance, args, kwargs):
         else:
             kwargs["interceptors"] = interceptors
         return wrapped(*args, **kwargs)
+
+
+class GRPCAsyncServerInstrumentation(AbstractInstrumentedModule):
+    name = "grpc_async_server_instrumentation"
+    creates_transactions = True
+    instrument_list = [("grpc", "aio.server")]
+
+    def get_instrument_list(self):
+        import grpc
```

Comment:
Can you please surround this with a `try:except` similar to [this example](https://github.com/elastic/apm-agent-python/blob/0eaa44a199365785c61ffe8a5adcf7b7a58a7a8d/elasticapm/instrumentation/packages/elasticsearch.py#L50-L62)?

Reply:
Ok, done! Should I also use this technique to check if the `grpc.aio` is present? Or just leave as I've done, with the version check?

Reply:
I think we're good. Anything in the `call()` will only be called if we decide to instrument. And anything in the instrument list that isn't able to import will be skipped.

Before 1.33.1, is `grpc.aio` just not able to be imported? Because now that I think about it, if we just set `instrument_list` to something that is only available in 1.33.1+, we can just let the instrumentation system attempt to import it and then abort.

If it's present in <1.33.1 but not working, then we need the version check as you've coded it. But if it's just missing, can probably skip the whole version check.

Reply:
I didn't actually check by testing or anything like that, but the release changelog says: "Allow asyncio API to be imported as grpc.aio.", so I'm guessing it actually didn't exist before that version. And the date is close to the first commit in the history of file `__init__.py` in the `grpc/aio` folder. So I think it's a good idea to skip the version check like you suggested! I'll push a commit removing it now.